### DISCUSSION TITLE
Phone number cleaning for Australian numbers?

### DISCUSSION THREAD

**Discussion by halloleo** [sfu-db/dataprep#980] [Q&A]
The [clean_phone()](https://docs.dataprep.ai/api_reference/dataprep.clean.html#dataprep.clean.clean_phone.clean_phone) function seems to work mainly for US/Canada numbers.

Is there any way to clean Australian phone numbers?

**Comment by qidanrui** [comment]
Thanks for your suggestion! We will support the Austrilian phone numbers later~
